TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/datahandlers/NettyPacketToHttpConsumerTest.java

```diff
@@ -315,6 +316,90 @@ public void testMetricCountsFor_testThatConnectionsAreKeptAliveAndShared(boolean
         Assertions.assertEquals(2, connectionsClosedCount);
     }
 
+    @ParameterizedTest
+    @CsvSource({"false", "true"})
+    @Tag("longTest")
```

Comment:
How long will this actually take to run?  It seems like it should be < 1s.

Reply:
The repettitions were making it take longer. I'll change that to 1 reps and it's 1 second for both tls and non-tls